app/scripts/views/settings/avatar_camera.js

```diff
@@ -7,7 +7,6 @@
 define([
   'underscore',
   'cocktail',
-  'canvasToBlob',
```

Comment:
We cannot remove this, polyfill for older browsers: https://github.com/mozilla/fxa-content-server/blob/cfe4db3b4e390725f430dc9127f7adbbb233311e/app/scripts/views/settings/avatar_camera.js#L200

Needs `exceptsPaths`